.github/workflows/publish-rust.yml

```diff
@@ -68,7 +68,7 @@ jobs:
         run: pnpm zx ./scripts/rust/test.mjs "${{ inputs.package_path }}"
 
       - name: Check semver
-        run: pnpm rust:semver ${{ inputs.package_path }} --release-type ${{ inputs.level }}
+        run: pnpm rust:semver --manifest-path "${{ inputs.package_path }}/Cargo.toml" --release-type ${{ inputs.level }}
```

Comment:
This will work for crates `> 0.x`. Not sure if you would like to update the script with the latest fix: https://github.com/anza-xyz/pinocchio/blob/main/.github/workflows/publish.yml#L52-L81

Reply:
Sure, done!